tsconfig.base.json

```diff
@@ -21,6 +21,7 @@
       "@vegaprotocol/cypress": ["libs/cypress/src/index.ts"],
       "@vegaprotocol/deal-ticket": ["libs/deal-ticket/src/index.ts"],
       "@vegaprotocol/deposits": ["libs/deposits/src/index.ts"],
+      "@vegaprotocol/environment": ["libs/environment/src/index.ts"],
```

Comment:
chore (blocking): Need to delete network-switcher lib

Reply:
right, missed this, removed now!